CHANGELOG

```diff
@@ -3,7 +3,7 @@
 ## [Unreleased]
```

Comment:
Should we create a new release for this? I think every push to GitHub should generally be a new release and have a new version number.

Reply:
Done.